pocs/filterwheel/filterwheel.py

```diff
@@ -137,6 +140,10 @@ def current_filter(self):
     def current_filter(self, filter_name):
         self.move_to(filter_name, blocking=True)
 
+    @property
+    def is_unidirectional(self):
+        return self._unidirectional
```

Comment:
I don't see `._unidirectional` defined anywhere.

Reply:
Yeah, I haven't set it in the abstract base class, but I have given it a value in all the derived classes, either in `__init__()` or `connect()`. Maybe better to have the method just raise `NotImplementedError` here, to make it more obvious that derived classes need to do something?